samples/snippets/src/main/java/pubsublite/CreateSubscriptionExample.java

```diff
@@ -83,7 +111,12 @@ public static void createSubscriptionExample(
 
     try (AdminClient adminClient = AdminClient.create(adminClientSettings)) {
       Subscription response = adminClient.createSubscription(subscription).get();
-      System.out.println(response.getAllFields() + "created successfully.");
+      if (regional) {
+        System.out.println(
+            response.getAllFields() + " (regional subscription) created successfully.");
+      } else {
+        System.out.println(response.getAllFields() + " (zonal subscription) created successfully.");
+      }
```

Comment:
Revert? I don't think this is necessary, the response will have the full regional or zonal path in it

Reply:
Reverted.